docs/workload-development-kit/extensibility-front-end.md

```diff
@@ -297,8 +297,8 @@ An example with `notification.open()` API:
 While a frontend-only development scenario is easily supported, the full end-to-end developer experience requires saving, reading, and editing existing workload items.
 The [Back-end implementation guide](extensibility-back-end.md) describes in detail how to set up and use the backend side.
 
-Once the backend is up and running, and the `Fabric.WorkloadSample.SampleWorkloadArtifact` type is **registered in Fabric**, you can perform CRUD operations on this type.
-The following operations are exposed via [ArtifactCrud API](https://github.com/microsoft/Microsoft-Fabric-developer-sample/tree/main/Frontend/node_modules/@trident/extension-client-3p/src/lib/apis/artifact-crud-api.d.ts) inside `ExtensionClientAPI`.
+Once the backend is up and running, and the `Org.WorkloadSample.SampleWorkloadItem` type is **registered in Fabric**, you can perform CRUD operations on this type.
+The following operations are exposed via [ArtifactCrud API](https://learn.microsoft.com/en-us/javascript/api/@ms-fabric/workload-client/artifactcrudapi?view=fabric-extensibility-js-latest).
```

Comment:
Always remove this part of the URL (if it exitsts): `https://learn.microsoft.com/en-us/`.

You also need to remove the question mark and everything that comes after it.

Replace:

https://learn.microsoft.com/en-us/javascript/api/@ms-fabric/workload-client/artifactcrudapi?view=fabric-extensibility-js-latest

With:

/javascript/api/@ms-fabric/workload-client/artifactcrudapi